src/main/java/org/springframework/data/elasticsearch/annotations/Score.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.data.elasticsearch.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Specifies that this field is used for storing the document score.
+ * 
+ * @author Sascha Woo
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+@Documented
+@Inherited
+public @interface Score {
```

Comment:
Let's meta-annotate this with `@ReadOnlyProperty` so that `@Transient` is not needed in the domain model as we always consider a score property metadata, right?